script/deploy-module.sh

```diff
@@ -0,0 +1,103 @@
+source .env
+
+set -e
+
+if [[ $1 == "" || $2 == "" || ($3 != "--verify-only" && $3 != "")]]
+    then
+        echo "Usage:"
+        echo "  deploy-module.sh [target environment] [contractName] --verify-only [constructor-args]"
+        echo "    where target environment (required): mainnet / testnet / sandbox"
+        echo "    where contractName (required): contract name you want to deploy"
+        echo "    --verify-only: if you only want to verify the existing deployment source code"
+        echo "                   [constructor-args] are ABI-Encoded for verification (see 'cast abi-encode' docs)"
+        echo ""
+        echo "Example:"
+        echo "  deploy-module.sh sandbox StepwiseCollectModule"
+        exit 1
+fi
+
+NETWORK=$(node script/helpers/readNetwork.js $1)
+if [[ $NETWORK == "" ]]
+    then
+        echo "No network found for $1 envirotnment target in addresses.json. Terminating"
+        exit 1
+fi
+
+if [[ $1 == "mainnet" ]]
```

Comment:
Will need to be changed if/when we support new networks, such that if a network is not recognized instead of defaulting to testnet, it should throw an error. 

Not a hard requirement, I think this is fine for now but I would still require explicit checking here; we could misspell mainnet for instance, too!